lich.rbw

```diff
@@ -10222,6 +10222,15 @@ unless File.exists?(SCRIPT_DIR)
       exit
    end
 end
+unless File.exists?("#{SCRIPT_DIR}/custom")
```

Comment:
Is this platform agnostic, or do we need to do whatever Ruby's version of `Path.Join(...)` is?

Reply:
I don't know, but I'll look into it.  I just used the same syntax that is used for all of the other lich folder checks and creation.

Reply:
To be clear, are you asking about `"#{SCRIPT_DIR}/custom"` or ``File.exists?

Reply:
Oh, oops.  Clearly the string portion given the suggestion of `Path.join`  I'll change it to use that.